src/client/extension.ts

```diff
@@ -115,7 +115,7 @@ export async function activate(
     const areCRUDoperationEnabled = vscode.workspace.getConfiguration(SETTINGS_EXPERIMENTAL_STORE_NAME).get(PORTAL_CRUD_OPERATION_SETTING_NAME);
     if (areCRUDoperationEnabled) {
```

Comment:
can we remove this FF now?

Reply:
sure - let me take that up in a separate PR